frontend/src/pages/login/components/CrazeeBurgerLogo.tsx

```diff
@@ -1,9 +1,13 @@
 import styled from 'styled-components'
-import { THEME } from '../../../reusable-ui/design-system'
+import { T } from '../../../reusable-ui/design-system'
```

Comment:
```suggestion
import { theme } from '../../../reusable-ui/design-system'
```
Privilégier les nommages explicites comme "theme".
Quand on voit T comme ça, on pense au T du typage dynamique.